tests/no-camelcase.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const rule = require( '../rules/no-camelcase' );
+const RuleTester = require( 'eslint' ).RuleTester;
+
+const error = '$.camelcase is not allowed';
```

Comment:
Looks like it should be camel**C**ase

Reply:
Fixed (and I also fixed tests to use the new RuleTesterAndDocs)